ci/util.py

```diff
@@ -332,6 +333,17 @@ def merge_dicts(base: dict, other: dict, list_semantics='merge'):
     return merger.merge(deepcopy(base), deepcopy(other))
 
 
+DISTRIBUTION_NAME = 'gardener-cicd-libs'
+
+
+def gardener_cicd_libs_version():
+    '''Return the version of the active gardener-cicd-libs distribution
+    '''
+    for distribution in pkg_resources.working_set:
```

Comment:
why doing this cumbersome lookup? there is already a sane way to access entries by key:
```
import pkg_resources as pr
ws = pr.working_set
ws.by_key['gardener-cicd-libs']

gardener-cicd-libs 1.417.0 (/usr/lib/python3.7/site-packages)
```
How I found out about this? Well, I looked briefly into the documentation, because it was hard for me to believe that this module does not expose any more convenient way for such a straight-forward query. Turns out it does.